plotly/Make_a_FigureFactory.md

```diff
@@ -0,0 +1,113 @@
+# Add Your Figure Factory to the Plotly [Python Library](https://plot.ly/python/)
+
+If you have ever wanted to contribute to the Plotly Python Library by adding a new chart type we don't have, now you have the resources to do so. This README will help you get started cloning the plotly.py repo, forking a new branch, creating a new figure factory, and pushing your results to the cloud to get feedback for merging. Just follow all these steps and you'll be ready to go.
+
+## Getting Started:
+1. Clone the `plotly.py` repo locally and then check out the master branch.
+
+```
+$ git clone git@github.com:plotly/plotly.py.git
+$ git fetch origin
+$ git checkout master
+```
+
+2. Checkout a new branch and give it an appropriate name.
+
+```
+$ git checkout -b "my-new-ff"
+```
+
+## Create A figure_factory file
+1. Creating python file and updating `__init__.py`
+
+You are now ready to start writing your code. Begin by moving to the `plotly.figure_factory` directory in the `plotly.py` repo. Notice that there is an `__init__.py` file as well as a bunch of `_figure_factory_chart.py` files in this directory. Each type of unique plotly chart gets its own python file, and the name of each python file is found in the `__init__.py` file.
```

Comment:
rm You are now ready to start writing your code.
The directions might be a little easier to follow if they're more straight forward (i.e. more step by step and less conversational)

Reply:
>Each type of unique plotly chart gets its own python file

That's not really true, not each type of unique plotly chart but rather each figure factory chart. I think this is important to distinguish so people don't think they can edit a non-figure factory chart type from this repo (and end up looking for the scatter python file for instance).

Reply:
i'll clarify, thank you